### PR TITLE
Adjust Structure

### PR DESCRIPTION
**Pull request by AurevoirXavier**

### CODE REVIEW

File: node/cli/Cargo.toml [hunk 1]
```diff
@@ -21,7 +21,7 @@ structopt = { version = "0.3.21", optional = true }
 # darwinia-network
 bsc-primitives = { path = "../../primitives/bsc" }
 dp-consensus   = { path = "../../primitives/consensus" }
-drml-service   = { path = "../service" }
+service        = { package = "drml-service", default-features = false, path = "../service" }
```
Comment (AsceticBear) [inline node/cli/Cargo.toml:24]: `default-features = false` is necessnary here?

Comment (AurevoirXavier) [reply node/cli/Cargo.toml:24]: Yes. Otherwise, we can not control its features.

Comment (AsceticBear) [reply node/cli/Cargo.toml:24]: Weird, why the old version can compile without `default-features = false`?

Comment (AurevoirXavier) [reply node/cli/Cargo.toml:24]: Because all the features set to enable by default.

**Review by AsceticBear — APPROVED** [review]
LGTM

**Pull request merged**